app/src/main/java/com/example/polina/socialnetwork/PostAdapter.java

```diff
@@ -0,0 +1,111 @@
+package com.example.polina.socialnetwork;
+
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.CheckBox;
+import android.widget.CompoundButton;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.android.volley.toolbox.ImageLoader;
+import com.android.volley.toolbox.Volley;
+
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Created by polina on 11.07.15.
+ */
+public class PostAdapter extends ArrayAdapter<Post> {
+    LayoutInflater layoutInflater;
+    Context context;
+
+    public PostAdapter(Context context, List<Post> objects) {
+        super(context, R.layout.post_list, objects);
+        this.context = context;
+        layoutInflater = (LayoutInflater) context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
+
+    }
+
+    @Override
+    public View getView(int position, View convertView, ViewGroup parent) {
+
+        View view = convertView;
```

Comment:
not effective. use ViewHolder pattern. http://developer.android.com/training/improving-layouts/smooth-scrolling.html
also, header should be part of scrollable list http://developer.android.com/reference/android/widget/ListView.html#addHeaderView(android.view.View)